packages/help-center/src/hooks/use-get-zendesk-fields-by-flow.ts

```diff
@@ -0,0 +1,21 @@
+import { isDIFMFlow, isHundredYearDomainFlow, isHundredYearPlanFlow } from '@automattic/onboarding';
+
+export function useGetZendeskFieldsByFlow( flowName: string ) {
+	const url = window.location?.href;
```

Comment:
I'm not sure if location can be null.

Reply:
I don't think indeed, I will remove the check.